apx-parameters.rst

```diff
@@ -21,18 +21,33 @@ Parameters recognized in configuration files under /etc/cvmfs:
 CVMFS_ALIEN_CACHE               If set, use an alien cache at the given location
 CVMFS_ALT_ROOT_PATH             | If set to *yes*, use alternative root catalog path.
                                 | Only required for fixed catalogs (tag / hash) under the alternative path.
+CVMFS_ARCH                      | Automatically set by CVMFs to reflect the CPU architecture on which the client runs (using ``uname -m``).
```

Comment:
We should do a separate section for automatically generated parameters (in a follow-up PR)

Reply:
so can we merge it as it is or do you want me to take it out?